docs/api/api_docs.yaml

```
@@ -64,124 +99,616 @@ paths:
       tags:
         - Admin
         - Public
+      parameters:
+        - $ref: '#/components/parameters/include_docs'
+        - $ref: '#/components/parameters/Include-channels'
+        - $ref: '#/components/parameters/include-access'
+        - $ref: '#/components/parameters/include-revs'
+        - $ref: '#/components/parameters/include-seqs'
+        - $ref: '#/components/parameters/keys'
+        - $ref: '#/components/parameters/startkey'
+        - $ref: '#/components/parameters/endkey'
+        - $ref: '#/components/parameters/limit'
   '/{db}/_bulk_docs':
     parameters:
       - $ref: '#/components/parameters/db'
     post:
       responses:
-        '200':
-          description: OK
+        '201':
+          description: |-
+            Executed all operations. 
+
+            Each object in the returned array represents a document. Each document should be checked to make sure it was successfully added to the database.
+          content:
+            application/json:
+              schema:
+                type: array
+                uniqueItems: true
+                items:
+                  type: object
+                  properties:
+                    id:
+                      type: string
+                      description: The ID of the document that the operation was performed on.
+                    rev:
+                      type: string
+                      description: The new revision of the document if the operation was a success.
+                    error:
+                      type: string
+                      description: The error type if the operation of the document failed.
+                    reason:
+                      type: string
+                      description: The reason the operation failed.
+                    status:
+                      type: string
+                      description: The HTTP status code for why the operation failed.
+                  required:
+                    - id
+              examples:
+                Success:
+                  value:
+                    - id: FooBar
+                      rev: 1-cd809becc169215072fd567eebd8b8de
+                    - id: AliceSettings
+                      rev: 6-b3e8dcf825b71ccee112f3572ec4323c
+                    - id: BobSettings
+                      rev: 2-5145e1086bb8d1d71a531e9f6b543c58
+                All operations fail due to conflict:
+                  value:
+                    - error: conflict
+                      id: FooBar
+                      reason: Document exists
+                      status: 409
+                    - error: conflict
+                      id: AliceSettings
+                      reason: Document revision conflict
+                      status: 409
+                    - error: conflict
+                      id: BobSettings
+                      reason: Document revision conflict
+                      status: 409
```

Would be nice to have an example that demonstrates a partial success.

```suggestion
                Partial success:
                  value:
                    - id: FooBar
                      rev: 1-cd809becc169215072fd567eebd8b8de
                    - id: AliceSettings
                      error: conflict
                      reason: Document revision conflict
                      status: 409
                    - id: BobSettings
                      rev: 2-5145e1086bb8d1d71a531e9f6b543c58
```